development/compiling/compiling_for_windows.rst

```diff
@@ -331,152 +130,122 @@ in C++ is needed, for creating modules or extending the engine, working
-| **Fedora**    | ::                                                     |
-|               |                                                        |
-|               |     dnf install scons mingw32-gcc-c++ mingw64-gcc-c++  |
-+---------------+--------------------------------------------------------+
-| **macOS**     | ::                                                     |
-|               |                                                        |
-|               |     brew install scons mingw-w64                       |
-+---------------+--------------------------------------------------------+
-| **Mageia**    | ::                                                     |
-|               |                                                        |
-|               |     urpmi scons mingw32-gcc-c++ mingw64-gcc-c++        |
-+---------------+--------------------------------------------------------+
-
-Before allowing you to attempt the compilation, SCons will check for
-the following binaries in your ``$PATH``:
-
-::
+If you are a Linux or macOS user, you need to install
+`MinGW-w64 <https://mingw-w64.org/doku.php>`_, which typically comes in 32-bit
+and 64-bit variants. The package names may differ based on your distribution,
+here are some known ones:
+
++---------------+--------------------------------------------------------------+
+| **Arch**      | ::                                                           |
+|               |                                                              |
+|               |     pacman -S mingw-w64-gcc                                  |
++---------------+--------------------------------------------------------------+
+| **Debian** /  | ::                                                           |
+| **Ubuntu**    |                                                              |
+|               |     apt install mingw-w64                                    |
++---------------+--------------------------------------------------------------+
+| **Fedora**    | ::                                                           |
+|               |                                                              |
+|               |     dnf install mingw64-gcc-c++ mingw64-winpthreads-static \ |
+|               |                 mingw32-gcc-c++ mingw32-winpthreads-static   |
++---------------+--------------------------------------------------------------+
+| **macOS**     | ::                                                           |
+|               |                                                              |
+|               |     brew install mingw-w64                                   |
++---------------+--------------------------------------------------------------+
+| **Mageia**    | ::                                                           |
+|               |                                                              |
+|               |     urpmi mingw64-gcc-c++ mingw32-gcc-c++                    |
```

Comment:
Could you add `mingw64-winpthreads-static and mingw32-winpthreads-static`? It turns out they are also needed.